src/main/fc/fc_core.c

```diff
@@ -627,9 +626,8 @@ static void subTaskMotorUpdate(void)
     }
 #endif
 
-    if (motorControlEnable) {
-        writeMotors();
-    }
+    writeMotors();
```

Comment:
Good point. It turns out there was already a guard in `pwmCompleteDshotMotorUpdate` - I've moved this up a level.